pyproject.toml

```diff
@@ -45,12 +46,13 @@ ansys-tools-filetransfer = ">=0.1"
 networkx = ">=3.0.0"
 
 # Optional dependencies which are included in the 'extras' section.
-# Update also the 'dev' group when these are updated.
+# TODO: Once ansys-mapdl-core and ansys-mechanical-core support Python 3.13,
+# remove the 'python' limit below and update the main Python version in CI.
 pyvista = { version = ">=0.42.0", optional = true, extras = ["jupyter", "trame"] }
-ansys-mapdl-core = { version = ">=0.68.3", optional = true }
+ansys-mapdl-core = { version = ">=0.68.3", optional = true, python=">=3.10,<3.13" }
 ansys-dpf-composites = { version = ">=0.6", optional = true }
 ansys-dpf-core = { version = ">=0.13", optional = true}
-ansys-mechanical-core = { version = ">=0.10.0", optional = true }
+ansys-mechanical-core = { version = ">=0.10.0", optional = true, python=">=3.10,<3.13" }
```

Comment:
To revert before merging, to avoid accidentally leaving them out (these are user-facing dependencies via the `examples` and `all` extras).